### PR TITLE
Add tests for gardener-extension-shoot-networking-filter.

### PR DESCRIPTION
**Pull request by axel7born**
/kind enhancement

**What this PR does / why we need it**:
As a broken networking filter could possibly remain unnoticed for a long time, it's important to test it on a regular basis.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: config/jobs/gardener-extension-shoot-networking-filter/gardener-extension-shoot-networking-filter-unit-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+presubmits:
+  gardener/gardener-extension-shoot-networking-filter:
+  - name: pull-extension-shoot-networking-filter
```
Comment (oliver-goetz) [inline config/jobs/gardener-extension-shoot-networking-filter/gardener-extension-shoot-networking-filter-unit-tests.yaml:3]:
```suggestion
  - name: pull-extension-shoot-networking-filter-unit
```

File: config/jobs/gardener-extension-shoot-networking-filter/gardener-extension-shoot-networking-filter-unit-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        args:
+        - verify-extended
+        resources:
+          limits:
+            memory: 2Gi
+          requests:
+            cpu: 2
+            memory: 1Gi
+periodics:
+- name: ci-extension-shoot-networking-filter
```
Comment (oliver-goetz) [inline config/jobs/gardener-extension-shoot-networking-filter/gardener-extension-shoot-networking-filter-unit-tests.yaml:26]:
```suggestion
- name: ci-extension-shoot-networking-filter-unit
```

**Review by oliver-goetz — COMMENTED** [review]
Thanks for the PR 🚀 

I have only one remark to keep the naming of the tests consistent.

**Review by ScheererJ — APPROVED** [review]

**Pull request merged**